nixos/modules/services/x11/xwayland.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xwayland;
```

Comment:
Since this is not strictly a service (in the running-as-a-daemon sense) it would more appropriate to put this option under the `programs` namespace and move the module to `nixos/modules/programs/`.

Reply:
Well, it depends on which definition of service we use:
- `Xwayland` could be run at boot time by the display manager
- `Xwayland` is running in the background, translating requests from X protocol to Wayland protocol (and vice-versa)
- `Xwayland` is actually `xorg-server` (which is considered as a service) compiled with specific configure options 

I've moved it to `programs`, however if you agree with the reasons I provided, I could move it back

Reply:
The file is still in `modules/services/x11`, but I thought that made sense because it's X11 stuff